wren-tests/src/test/java/io/wren/testing/bigquery/AbstractWireProtocolTestWithBigQuery.java

```diff
@@ -88,16 +83,4 @@ protected String getDefaultSchema()
     {
         return "tpch_tiny";
     }
-
-    @Override
-    protected void cleanup()
-    {
-        try {
-            BigQueryMetadata metadata = getInstance(Key.get(BigQueryMetadata.class));
-            BigQueryClient bigQueryClient = metadata.getBigQueryClient();
```

Comment:
There're are a bug. We will create a pg_catalog in BigQuery. We should remove it after test. 
```
bigQueryClient.dropDatasetWithAllContent(DatasetId.of(getDefaultCatalog(), metadata.getPgCatalogName()));
```

Reply:
It was lost due to #528. I will add a commit to fix it.